yarn.lock

```diff
@@ -2814,6 +2814,47 @@
     secp256k1 "4.0.2"
     sha3 "2.1.4"
 
+"@hyperledger/cactus-common@1.0.0":
```

Comment:
@RafaelAPB Let's wait for the fix to the substrate test ledger (hopefully in v1.1.3) and that way we can avoid adding an old dependency to the lock file like the v1.0.0 test tooling package (could lead to interesting build failures and confusion).

Please:
1. Create a bug in the issue tracker for the substrate test ledger problem we discovered the morning of the workshop
2. Mark this PR dependent on that.
3. That way when that issue gets fixed we'll know that this one is ready for prime time (at which time I'll ask you to bump the test-tooling package dependency version to the one with the fix)

Reply:
Done. Could we instead accept the PR with test-tooling version 1.0.0 (one that passes), so that we can use the code on the PT workshop, next week? @petermetz